src/main/flight/imu.c

```diff
@@ -749,6 +749,13 @@ void imuUpdateAttitude(timeUs_t currentTimeUs)
 }
 #endif // USE_ACC
 
+// Angle in between the nose axis of the craft and the horizontal plane in ground reference.
+// Poitive angle - nose down, negative angle - nose up.
```

Comment:
(positive)

Is sign of value correct? IIRC `rMat[?][0]` is projection of body X axis into earth frame, `rMat[2][0]` is Z component of it. With BF coordinate system, I'd expect it to be positive when pitching up (but BF coordinate system are sometimes a bit misleading).

Also, in BF, `pitch = asin(-rMat[2][0]);`. 
Because pitch is second angle, pitch and dive are equivalent (this won't be true for roll and projection of Y axis). `getSinPitchAngle` name will work in this case.

Reply:
you are right, this is just pitch angle in IMU. I will double check the sign.
In this case, do you think it's better to just use `sin(attitude.values.pitch)` or it's worth to save a little performance, and still use `getSinPitchAngle()`?